docs/core_docs/.gitignore

```diff
@@ -29,3 +29,9 @@ yarn-error.log*
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+.quarto/
+
+
+# notebooks
+docs/modules/agents/agent_types/openai_assistant_2.md
```

Comment:
Can we just move all existing `.md` files to `.mdx` so that we don't need to individually add these?